### ISSUE TITLE
Downstream SAINTexpress non-zero exit code

### ISSUE THREAD

**Issue by FlyOB** [Nesvilab/FragPipe#1632]
**- Upload your log file**
(If a log file hasn't been generated, go to the 'Run' tab in FragPipe, click 'Export Log', zip the resulting "log_[date_time].txt" file to avoid truncation, then attach the zipped file by drag & drop here.)
[log_2024-06-14_10-09-21.zip](https://github.com/user-attachments/files/15839167/log_2024-06-14_10-09-21.zip)
[Downstream_log_2024-06-14_10_10_45.zip](https://github.com/user-attachments/files/15839253/Downstream_log_2024-06-14_10_10_45.zip)

**- Describe the issue or question:**

I successfully ran 9 DDA bio-replicates through FragPipe, but when attempting to run downstream SAINTexpress, it failed to finish creating necessary documents because of a "exit code:3". Not sure how to fix this.
Thanks.

**Comment by fcyu** [comment]
Could you rename your experiment group name to follow SAINT's requirement? I think the control group needs to be named to `CONTROL`.

@anesvi Do we have a tutorial or document somewhere in GitHub? I only found it in the short course material.

Best,

Fengchao

**Comment by FlyOB** [comment]
I have two types of control groups, do they need to be named differently to discriminate between them?

**Comment by fcyu** [comment]
I guess one SAINT analysis can only have one control groups? @guoci 

Best,

Fengchao

**Comment by FlyOB** [comment]
You are correct. I used CONTROL for my empty vector and used other identifiers for everything else and SAINTexpress worked. Thanks for the help.
